src/app/plugins/register.plugin.ts

```diff
@@ -17,16 +17,46 @@ export class RegisterPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args?: string[]) {
-    const request = this.container.classService.buildRequest(message.author, args);
-    if (!request) {
-      message.reply('I was unable to build your request.');
+    if (!args) {
       return;
     }
-    try {
-      const response = await this.container.classService.register(request);
-      message.reply(response);
-    } catch (e) {
-      message.reply(e);
+
+    const successfulClasses: string[] = [];
+    const invalidClasses: string[] = [];
+    let error: Maybe<string> = undefined;
+    for (const arg of args) {
+      const request = this.container.classService.buildRequest(message.author, [arg]);
+      if (!request) {
+        invalidClasses.push(arg);
+        continue;
+      }
+      try {
+        const response = await this.container.classService.register(request);
+        if (response.includes('success')) {
+          successfulClasses.push(arg);
+        } else {
+          invalidClasses.push(arg);
+        }
+      } catch (e) {
+        invalidClasses.push(arg);
+        error = e;
```

Comment:
just log the error with logger service. dont expose the actual error to the user

Reply:
I can, but before, it just replied to the user with the error

Reply:
you shouldnt expose the actual exception to the user

Reply:
Ok, should I log each error just in case, or just do one log with an error per call

Reply:
just log the error when it occurs that's all